type-definitions/ts-tests/empty.ts

```diff
@@ -1,57 +1,44 @@
+import { expect, test } from 'tstyche';
 import { Seq, Collection } from 'immutable';
 
-{
-  // Typed empty seqs
+test('typed empty Seq', () => {
+  expect(Seq()).type.toEqual<Seq<unknown, unknown>>();
 
-  // $ExpectType Seq<unknown, unknown>
-  Seq();
+  expect(Seq<number, string>()).type.toEqual<Seq<number, string>>();
 
-  // $ExpectType Seq<number, string>
-  Seq<number, string>();
+  expect(Seq.Indexed()).type.toEqual<Seq.Indexed<unknown>>();
 
-  // $ExpectType Indexed<unknown>
-  Seq.Indexed();
+  expect(Seq.Indexed<string>()).type.toEqual<Seq.Indexed<string>>();
 
-  // $ExpectType Indexed<string>
-  Seq.Indexed<string>();
+  expect(Seq.Keyed()).type.toEqual<Seq.Keyed<unknown, unknown>>();
 
-  // $ExpectType Keyed<unknown, unknown>
-  Seq.Keyed();
+  expect(Seq.Keyed<number, string>()).type.toEqual<Seq.Keyed<number, string>>();
 
-  // $ExpectType Keyed<number, string>
-  Seq.Keyed<number, string>();
+  expect(Seq.Set()).type.toEqual<Seq.Set<unknown>>();
 
-  // $ExpectType Set<unknown>
-  Seq.Set();
+  expect(Seq.Set<string>()).type.toEqual<Seq.Set<string>>();
```

Comment:
Interestingly `dtslint` does not see difference between `Set` and `Seq.Set`. Probably this is how it works. I did not know this detail. Learned something new.

Reply:
Indeed it is 😄